source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4121,10 +4125,6 @@ export class ToggleInput extends Input {
             this._checkboxInputElement.setAttribute("aria-label", this.title);
         }
 
-        if (this.isRequired) {
```

Comment:
why do you remove this?

Reply:
I added `aria-required` the parent input renderer. This assignment should now be redundant.

I need to verify this with the CI site.

Reply:
Updated